### PR TITLE
Update changelog with recent work.

### PR DESCRIPTION
**Pull request by patrickvacek**
Hopefully if you are happy I can make the release tomorrow!

### CODE REVIEW

**Comment by lbonn** [comment]
Seems ok but would advise to wait until the merge of https://github.com/advancedtelematic/aktualizr/pull/1192

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,9 +6,33 @@ Our versioning scheme is `YEAR.N` where `N` is incremented whenever a new releas
...
+### Removed
+
+- example.com is no longer set as the default URL when using garage-sign via garage-deploy: [PR](https://github.com/advancedtelematic/aktualizr/pull/1169)
+- OPC-UA secondary support: [PR](https://github.com/advancedtelematic/aktualizr/pull/1177)
+
+### Fixed
+
+- Check for updates even if sending the manifest fails: [PR](https://github.com/advancedtelematic/aktualizr/pull/1186)
+- Correctly handle empty targets metadata: [PR](https://github.com/advancedtelematic/aktualizr/pull/1186)
+- Various OSTree-related memory leaks: [PR #1114](https://github.com/advancedtelematic/aktualizr/pull/1114), [PR #1120](https://github.com/advancedtelematic/aktualizr/pull/1120), and [PR #1179](https://github.com/advancedtelematic/aktualizr/pull/1179)
```
Comment (lbonn) [inline CHANGELOG.md]: minor: #1179 is technically not a leak fix but merely an update of the suppression list (for supposed false positives) for the test suite.

Comment (patrickvacek) [reply CHANGELOG.md]: True. I can update it once we we figure out #1192.

Comment (patrickvacek) [reply CHANGELOG.md]: Done and now ready for review.

**Comment by lbonn** [comment]
Should we mention the work on Android? https://github.com/advancedtelematic/aktualizr/pull/1034

**Comment by patrickvacek** [comment]
> Should we mention the work on Android?

Sure! Done. And rebased.

**Pull request merged**